Mapchat/app/src/main/res/values/google_maps_api.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
     <string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">
-        AIzaSyDvLJAYzTuexV9ReXRCwCwFkYonmwvO418
+        AIzaSyAca5KFPJZwNMo349jdAq-dtE8-yyF0Zvc

```

Comment:
Invalidated this key. Creating a new key, ignoring from git and will send privately